decidim-elections/lib/decidim/votings/census.rb

```diff
@@ -5,6 +5,12 @@ module Votings
     # This module contains all the domain logic associated to Decidim's Votings
     # Census.
     module Census
+      include ActiveSupport::Configurable
+
+      # How long the census access codes export file will are available in the server
+      config_accessor :census_access_codes_export_expiry_time do
+        2.days
```

Comment:
Is this something that we want to be configurable in the future? We could add a setting to the participatory space.

Reply:
💬 Mmm, I think it's ok to leave it only here. This is a setting that affects security, so I'd leave it to be changed only by system administrators, not users.